tests/conftest.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 Canonical Ltd.
+# Licensed under the GPLv3, see LICENCE file for details.
```

Comment:
The LICENCE file seems to be an Apache 2.0 licence? In other repos we have been making the second line just `see LICENCE file for details.` to make it easier to maintain